### DISCUSSION TITLE
Sync interupted by Windows update, and it cause a BUG

### DISCUSSION THREAD

**Discussion by cccat6** [Chia-Network/chia-blockchain#2090] [Q&A]
I fired up a windows machine just for farming. I install the Chia GUI and waiting for it to sync. However, in the process of syncing, it was interrupted by Windows update. Once I reopen the Chia GUI, a bug happened.
Initially, the status is "Not Synced". Then It will turn to this pic:
![image](https://user-images.githubusercontent.com/22387156/115185006-928a8680-a111-11eb-80b2-5a1e7151a3af.png)
Which is 24001 out of 0. A few seconds later, it will change to:
![image](https://user-images.githubusercontent.com/22387156/115185056-ab933780-a111-11eb-85ad-613da29cc6b7.png)
Which seems correct, right? But a few seconds later, it goes back to "Not Synced"
![image](https://user-images.githubusercontent.com/22387156/115185172-e6956b00-a111-11eb-8116-971cabef5be5.png)
I tried to reinstall Chia, but I don't know where the Nodes file and my key files are stored. After reinstall, the thing doesn't change. Is it necessary to clear the whole system?
Since the sync process is such slow. Is it possible that I copy my synced Nodes file to the other machine?

**Comment by cccat6** [comment]
I find the file. It is in the User dir .chia/mainnet
It is too large to copy anyway (about 2.13GB for now, which is Apr. 19 2021).
I simply delete it and let it sync from 0.
Will the full node compress in the future? It might get larger and larger as time run and spend more and more time syncing.

**Reply by cccat6** [reply]
The version of Chia is 1.0.5

**Reply by InfinitePotential2020** [reply]
My Machine has been running for 4 days straight. I checked and still no new XCH..
Connection Status is Connected, Status is Not Synced..
I'm running 1.0.5.
Why is that??
Thanks.

**Reply by cccat6** [reply]
I guess the problem you have is "Not Synced".
Try to uninstall Chia and delete all the blocks data here `User dir .chia/mainnet`
It will spend about a day to sync all the blocks for now. You should expect to see the progress which shows "Syncing".

**Comment by chriswong23** [comment]
It happens to me on Windows GUI running on version 1.0.5 as well. After I restart my machine, it sometimes get stuck at Not Synced even if I leave it for a day. I can't see any connection being made to any other node. I've tried to re-install it and sync from 0. It works for a while and after getting fully synced, this problem pops up randomly if I close and reopen the app. To get around this, I manually connect the Windows Chia app to a RaspPi Chia node I have running as a Farmer on my local network on port 8444. That farmer doesn't experience this problem no matter how many times I restart that machine. I wonder if this is a Windows bug?
